### PR TITLE
NAPPS-1592: fix invalid arn error on deploy

### PR DESCRIPTION
**Pull request by Kat-Alo**
Getting this error in the scheduled ECS stack on deploy: 

```Parameter arn:aws:ecs:us-east-1:912288704264:task-definition/klaxon-scheduled-task-dev:2 is not valid. Reason: Provided Arn is not in correct format.```

We believe based on [the Events Rule Cloudformation docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-target.html) that this is because the Target ARN should be a cluster, not the task definition.

### CODE REVIEW

File: cfn/templates/ecs-task.template.yml [hunk 1]
```diff
@@ -149,27 +149,21 @@ Resources:
...
       ScheduleExpression: cron(0/10 * * * ? *)
       State: ENABLED
       Targets:
-        - Arn: !Ref KlaxonPageCheckTaskDefinition
+        - Arn: arn:aws:ecs:us-east-1:912288704264:cluster/newsroom-dev # cluster arn, hardcoding for now
           Id: klaxon-page-check-ecs-task
+          RoleArn: !GetAtt EC2ServiceEventsExecutionRole.Arn #ecs events role
+          EcsParameters:
+            TaskCount: 1
+            TaskDefinitionArn: !Ref KlaxonPageCheckTaskDefinition
```
Comment (aaronbrezel) [inline cfn/templates/ecs-task.template.yml:162]: We're adjusting the target in order to closely align with event rule [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-target.html) example: 
```
MyEventsRule:
  Type: AWS::Events::Rule
  Properties:
    Description: Events Rule with EcsParameters
    EventPattern:
      source:
        - aws.ec2
      detail-type:
        - EC2 Instance State-change Notification
      detail:
        state:
          - stopping
    ScheduleExpression: rate(15 minutes)
    State: DISABLED
    Targets:
      - Arn: !GetAtt 
          - MyCluster
          - Arn
        RoleArn: !GetAtt 
          - ECSTaskRole
          - Arn
        Id: Id345
        EcsParameters:
          TaskCount: 1
          TaskDefinitionArn: !Ref MyECSTask
```

Comment (aaronbrezel) [reply cfn/templates/ecs-task.template.yml:162]: Note how we must target a cluster as well as a ecs task

File: cfn/templates/ecs-task.template.yml [hunk 2]
```diff
@@ -149,27 +149,21 @@ Resources:
...
 
   # Role that allows EventBridge to schedule an ECS task to run
   # Used for Klaxon
-  ECSTaskExecutionRole:
+  EC2ServiceEventsExecutionRole:
```
Comment (aaronbrezel) [inline cfn/templates/ecs-task.template.yml:166]: renaming for clarity. This is the role that allows eventbridge to actually schedule the task. `TaskDefintionExecutionRole`, the other role defined in this file--is the role the task actually assumes as it's executing

**Review by paigemoody — APPROVED** [review]

**Review by aaronbrezel — APPROVED** [review]
Let's give it a rip

**Pull request merged**